pkg/apis/credentialprovider/BUILD

```diff
@@ -0,0 +1,37 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "doc.go",
+        "register.go",
+        "types.go",
+        "zz_generated.deepcopy.go",
+    ],
+    importmap = "k8s.io/cloud-provider-gcp/cmd/auth-provider-gcp/apis/credentialprovider",
+    importpath = "k8s.io/kubelet/pkg/apis/credentialprovider",
```

Comment:
This seems odd.

Reply:
Definitely a discrepancy there - removed the importmap.